news/2017-07-20-aspire-2017-osutaiko-voting-opens-soon.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Aspire 2017: osu!taiko Voting Opens Soon"
+permalink: http://osunews.tumblr.com/post/163212166448/aspire-2017-osutaiko-voting-opens-soon
+date: 2017-07-20 13:25:00 +0000
+tumblr_url: http://osunews.tumblr.com/post/163212166448/aspire-2017-osutaiko-voting-opens-soon
+---
+
+After a long period of preparation, Aspire 2017: osu!taiko is ready to begin! Join us this **Saturday the 22nd July at 14:00 UTC** for a livestreamed preview of the final twelve entries, complete with the commentary of our expert judging panel - mangomizer, Unmei Muma, and Nwolf!
+
+![](https://assets.ppy.sh/contests/47/header.jpg)
+
+We've been absolutely blown away by how visually stunning some of the entries are, with many of osu!taiko's finest creative minds pulling out all the stops to produce some truly spectacular displays.
+
+There will be carnage, there will be crazy storyboards, and there will be osu!taiko like you've never seen it done before. Even if you're not an avid player of osu!taiko, tune in and have a look!
+
+We'll be starting at **Saturday 22 July, 14:00 UTC** over at the [official osu! Twitch channel, osulive.](https://twitch.tv/osulive).
+
+Be there, or be square. Or as Evrien once put it, don't be square, because we are circles. 
+
+The voting will go live the moment the livestream preview begins. Vote along with Twitch chat if you like!
+
+Hope to see you there!
```

Comment:
these last two sentences just seem like "i have no idea how to end this" and got awkward but I actually don't have anything that sounds a bit less dumb

maybe "The voting will go live the moment the stream preview begins so you can vote along with Twitch chat and the commentary!"

Wouldnt have very last sentence say Hope to see you there but rather just assume people will come and take out the hope if that makes sense